lib/Doctrine/Adapter/Exception.php

```diff
@@ -31,4 +31,9 @@
  * @author      Konsta Vesterinen <kvesteri@cc.hut.fi>
  */
 class Doctrine_Adapter_Exception extends Doctrine_Exception
-{ }
\ No newline at end of file
+{
+    /**
+     * @var array
+     */
+    public $errorInfo;
```

Comment:
`PDOException` has this property, and there's a case where an Exception object could be `PDOException` or this class, so adding this property to make it compatible in that case.